testing/load-aware/config.yaml

```diff
@@ -102,17 +107,21 @@ clusters:
   cleanup_on_exit: false
 load_aware:
   log_level: 1
-  plugin: "TargetLoadPacking"
-  default_requests_cpu: "2000m"
+  plugin: TargetLoadPacking
+  default_requests_cpu: 2000m
   default_target_requests_multiplier: "2"
```

Comment:
I think this should be an int, no?

Reply:
Still not sure why, but looks like its supposed to be a string according to the source: https://github.com/kubernetes-sigs/scheduler-plugins/blob/6ef6f63a0826b6045dd955eac599c902e396457f/apis/config/types.go#L102C28-L102C34